src/main/kotlin/mate/academy/LetterRemover.kt

```diff
@@ -1,5 +1,14 @@
 package mate.academy
 
-class LetterRemover {
-    // implement ...
+object LetterRemover {
+        @JvmStatic
+        fun removeLetter(input: String): CharArray {
+            val characters : MutableList<Char> = mutableListOf()
```

Comment:
Please check this documentation
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/replace.html